### PR TITLE
Ensure operator role is allowed to stop instances

### PR DESCRIPTION
**Pull request by sam-nguyen7**
Using strings.Contains() instead of strings.HasPrefix() to properly catch operator roles that are allowed to stop instances.

### CODE REVIEW

File: pkg/services/chgm/chgm.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func isUserAllowedToStop(username, issuerUsername string, userDetails CloudTrail
 	for _, operatorIamName := range operatorIamNames {
 		// I added a '-' in the end to make sure it's the same as close as I can to the aws username (cannot guess the uuid)
 		operatorIamName = fmt.Sprintf("%s-%s-", infraID, operatorIamName)
-		if strings.HasPrefix(username, operatorIamName) {
+		if strings.Contains(username, operatorIamName) {
```
Comment (typeid) [inline pkg/services/chgm/chgm.go:98]: **Question:** Why is this change necessary? Did the formatting for the username change, or did this never work to begin with?

Comment (sam-nguyen7) [reply pkg/services/chgm/chgm.go:96]: some STS role names do not conform to the infraID-operatorIamName format, so we were not properly escalating some CHGM before
